templates/nextcloud28-deps.nomad

```diff
@@ -0,0 +1,165 @@
+        MYSQL_USER = "nextcloud28"
+        {{ with secret "liquid/nextcloud28/nextcloud.maria" }}
+          MYSQL_PASSWORD = {{.Data.secret_key | toJSON }}
+        {{ end }}
+        EOF
+        destination = "local/maria.env"
+        env = true
+      }
+      resources {
+        cpu = 200
+        memory = 250
+        network {
+          mbits = 1
+          port "maria" {
+            static = 8769
+          }
+        }
+      }
+      service {
+        name = "nextcloud28-maria"
+        port = "maria"
+        tags = ["fabio-:${config.port_nextcloud28_maria} proto=tcp"]
+        check {
+          name = "tcp"
+          initial_status = "critical"
+          type = "tcp"
+          interval = "${check_interval}"
+          timeout = "${check_timeout}"
+        }
+      }
+    }
+  }
+
+  group "minio-nextcloud28" {
+    ${ continuous_reschedule() }
+    ${ group_disk() }
+
+    task "minio-nextcloud28" {
+      ${ task_logs() }
+
+      constraint {
+        attribute = "{% raw %}${meta.liquid_volumes}{% endraw %}"
+        operator = "is_set"
+      }
+
+      driver = "docker"
+      config {
+        image = "${config.image('minio')}"
+        entrypoint = ["/bin/sh", "-c"]
+        args = ["mkdir -p /data/nextcloud && /usr/bin/docker-entrypoint.sh minio server --console-address :9001 /data"]
+        port_map {
+          s3 = 9000
+          console = 9001
+        }
+        labels {
+          liquid_task = "nextcloud28-minio"
+        }
+        memory_hard_limit = 5000
+        volumes = [
+          "{% raw %}${meta.liquid_volumes}{% endraw %}/nextcloud28/minio:/data",
+        ]
+      }
+
+      env {
+        MINIO_UPDATE = "off"
+        MINIO_API_REQUESTS_MAX = "600"
+        MINIO_API_REQUESTS_DEADLINE = "2m"
+        GOMAXPROCS = "20"
+      }
+
+      resources {
+        memory = 1500
+        cpu = 1000
+        network {
+          mbits = 1
+          port "s3" { static = 9005 }
+          port "console" { static = 9006 }
+        }
+      }
+
+      template {
+        data = <<EOF
+          {{- with secret "liquid/nextcloud28/minio.user" }}
+              MINIO_ROOT_USER = {{.Data.secret_key | toJSON }}
+          {{- end }}
+          {{- with secret "liquid/nextcloud28/minio.password" }}
+              MINIO_ROOT_PASSWORD = {{.Data.secret_key | toJSON }}
+          {{- end }}
+        EOF
+        destination = "local/minio.env"
+        env = true
+      }
+
+      service {
+        name = "nextcloud28-minio-s3"
+        port = "s3"
+        tags = ["fabio-:${config.port_nextcloud28_minio} proto=tcp"]
+
+        check {
+          name = "tcp"
+          initial_status = "critical"
+          type = "tcp"
+          interval = "${check_interval}"
+          timeout = "${check_timeout}"
+        }
+      }
+
+      service {
+        name = "nextcloud28-minio-console"
+        port = "console"
+        tags = ["fabio-/nextcloud28_minio_console strip=/nextcloud28_minio_console"]
```

Comment:
do we really care about hosting the minio console UI on fabio?  have you successfully used it? I think the `/nextcloud28_minio_console` prefix we put on the URL will cause it to break (as will most web apps that you change the root url of)
\